projects/storefrontlib/src/lib/product/components/product-list/container/product-list.component.ts

```diff
@@ -86,6 +90,24 @@ export class ProductListComponent implements OnChanges, OnInit {
   }
 
   ngOnInit() {
+    this.subscriptions
```

Comment:
tactical comment

Reply:
I'd prefer to avoid the explicit subscribe in the component logic and rather rely on the async pipe only. I can see the "price" (complexity) or multiple observables (i.e. for the brandcode only). What about a single async pipe that would load a UI specific model? We could combine multiple observables.